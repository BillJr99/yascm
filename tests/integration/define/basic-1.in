(define x 10)
x
(define y x)
y
(define (y) x)
(y) 
(define (x) -9223372036854775808)
(x)
(define y (x))
y
(define y x)
(y)
