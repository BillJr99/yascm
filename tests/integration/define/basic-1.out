10
10
10
-9223372036854775808
-9223372036854775808
-9223372036854775808
